R/derive_extreme_records.R

```diff
@@ -11,9 +11,12 @@
 #'
```

Comment:
Should we say here that `dataset` can be left `NULL`, in which case a dataset will be created? It initially confused me that there was no `dataset` in some of the examples, for instance the section "Flagging existence of source records (exist_flag, true_value, false_value)"

Reply:
Done